include/io.hpp

```diff
@@ -142,11 +144,21 @@ namespace NP {
 			in >> sus_max;
 		}
 
+		next_field(in);
+		if (more_fields_in_line(in))
+		{
+			in >> signal_type;
+			std::transform(signal_type.begin(), signal_type.end(), signal_type.begin(), [](unsigned char c){ return std::tolower(c); });
+			if (signal_type == "start") signal_at_completion = false;
```

Comment:
ok. Maybe we can consider using short-hand notations instead of full words to limit risk of typos for instance: 
`s` for start, `f` for finish, `x` for exclusion, ...